refs/biblio.json

```diff
@@ -456,7 +456,7 @@
         "authors": [
             "Bert Bos"
         ],
-        "href": "https://www.w3.org/Style/Group/css3-src/css3-positioning/Overview.html",
+        "href": "https://www.w3.org/TR/css-position-3/",
```

Comment:
This URL is already used by the auto-generated W3C reference in refs/w3c.json. As Specref disallows duplicate URLs, what you want to do here instead is to make this reference an alias to the existing W3C one, so basically replacing the whole of the object with:

```json
{
    "aliasOf": "css-position-3"
}
```

More on how Specref handles aliasing here: https://github.com/tobie/specref#aliases